writing-apps/our-first-app/the-build-system.md

```diff
@@ -69,6 +69,17 @@ If all went well, you should now be able to open your app from the Applications
 If you were about to add the "build" folder to your git repository and push it, stop! This binary was built for your computer and we don't want to redistribute it. In fact, we built your app in a separate folder like this so that we can easily delete or ignore the "build" folder and it won't mess up our app's source code.
 {% endhint %}
 
+## Uninstalling the application
+
+To uninstall your application, change into the build directory and we will use `ninja` once again to uninstall:
+
+```bash
+cd build
+sudo ninja uninstall
```

Comment:
You don't need to change to the build directory if using `ninja -C`:


```suggestion
sudo ninja -C build uninstall
```

The same is valid for the build and install lines.